doc/deployment.rst

```diff
@@ -92,9 +92,62 @@ terminal.  It should look something like this.
 These are two handy tools for debugging the configuration of your
 bus.
 
+Branching out to two machines
+-----------------------------
+
+Everything is tied together in fedmsg by the :term:`endpoints` dict.  It lets
+
+- A publishing service know what port it should be publishing on.
+- A consuming service know where the publisher is so it can connect there.
+
+Let's say you have two machines ``hostA`` and ``hostB``.  If you installed that
+fedmsg-relay on ``hostA`` as discussed above, then the config file in
+``/etc/fedmsg.d/relay.py`` is going to have values like
+``tcp://127.0.0.1:4001``.  That address will only work for local connectivity.
+Try changing *all* occurences of ``127.0.0.1`` in that file to ``hostA`` so
+that it looks something like this:
+
+.. code-block:: python
+
+    config = dict(
+        endpoints={
+            "relay_outbound": [
+                "tcp://hostA:4001",
+            ],
+        },
+        relay_inbound=[
+            "tcp://hostA:2003",
+        ],
+    )
+
+To confirm that something's not immediately broken, you can go through the
+tests of doing ``fedmsg-logger`` and ``fedmsg-tail`` on ``hostA`` again (all
+"local").
+
+Copy that relay.py file over to ``hostB`` with ``scp /etc/fedmsg.d/relay.py
+hostB:/etc/fedmsg.d/relay.py``
+
+You should now be able to run ``fedmsg-tail`` on ``hostA`` and have it receive
+a message from ``fedmsg-logger`` on ``hostB`` and vice versa have a
```

Comment:
Not in this case.  ``hostB`` knows to just send messages to the ``fedmsg-relay`` running on ``hostA``.